workers/tests/integrationtests/fixtures/vcr_cassettes/test_indicator_tags_ratio.json

```diff
@@ -1,6 +1,226 @@
+                "headers": {
+                    "host": [
+                        "api.ohsome.org"
+                    ],
+                    "accept": [
+                        "*/*"
+                    ],
+                    "accept-encoding": [
+                        "gzip, deflate"
+                    ],
+                    "connection": [
+                        "keep-alive"
+                    ],
+                    "user-agent": [
+                        "python-httpx/0.17.1"
+                    ],
+                    "content-length": [
+                        "958"
+                    ],
+                    "content-type": [
+                        "application/x-www-form-urlencoded"
+                    ]
+                }
+            },
+            "response": {
+                "status_code": 404,
+                "http_version": "HTTP/1.1",
+                "headers": {
+                    "Date": [
+                        "Wed, 19 May 2021 13:51:14 GMT"
+                    ],
+                    "Server": [
+                        "Apache/2.4.29 (Ubuntu)"
+                    ],
+                    "Content-Length": [
+                        "277"
+                    ],
+                    "Keep-Alive": [
+                        "timeout=5, max=100"
+                    ],
+                    "Connection": [
+                        "Keep-Alive"
+                    ],
+                    "Content-Type": [
+                        "text/html; charset=iso-8859-1"
+                    ]
+                },
+                "content": "<!DOCTYPE HTML PUBLIC \"-//IETF//DTD HTML 2.0//EN\">\n<html><head>\n<title>404 Not Found</title>\n</head><body>\n<h1>Not Found</h1>\n<p>The requested URL was not found on this server.</p>\n<hr>\n<address>Apache/2.4.29 (Ubuntu) Server at api.ohsome.org Port 443</address>\n</body></html>\n"
+            }
+        },
+        {
+            "request": {
+                "method": "POST",
+                "uri": "https://api.ohsome.org/v1elements/length/ratio",
```

Comment:
something went wrong there

Reply:
This was already fixed. But it seems like I forgot to recreate the VCR cassette.